WindowsAzure/Blob/Models/CreateBlobOptions.php

```diff
@@ -175,6 +185,28 @@ public function setBlobContentLanguage($blobContentLanguage)
     }
     
     /**
+     * Gets blob ContentLength.
+     *
+     * @return integer.
+     */
+    public function getBlobContentLength()
+    {
+        return $this->_blobContentLength;
+    }
+
+    /**
+     * Sets blob ContentLength.
+     *
+     * @param integer $blobContentLength value.
+     *
+     * @return none.
+     */
+    public function setBlobContentLength($blobContentLength)
+    {
+        $this->_blobContentLength = blobContentLength;
```

Comment:
See the pattern for sequenceNumber:
Validate::isInteger($blobContentLength, 'blobContentLength'); 
$this->_blobContentLength = $blobContentLength;